R/data.R

```diff
@@ -1010,3 +1010,19 @@
 #'   \item{label}{Cytoband label.}
 #' }
 "cytobands_hg38"
+
+#' DLBCL90 genes
+#'
+#' A data frame with genes and their weights for DLBCL90.
+#'
+#' @format ## `dlbcl90_genes`
+#' A data frame with 73 rows and 6 columns
+#' \describe{
+#'   \item{Gene.symbol}{Human-readable gene symbol.}
```

Comment:
Does it make sense to add a comment as to why there are two nearly identical gene symbol columns? Maybe: 

```
{Gene.symbol}{Human-readable gene symbol matching the symbols used for the DLBCL90 code set design.} 
...
{hgnc_symbol}{Human-readable gene symbol matching Gencode 33.}
```

Reply:
Thanks! Added this and pushed the changes! ✅